app/containers/Forms/ProductField.tsx

```diff
@@ -27,14 +27,13 @@ export const ProductFieldComponent: React.FunctionComponent<Props> = (
 ) => {
   const {formData, query, onChange} = props;
 
-  // TODO: Clean up this function with array.some() as noted in https://github.com/bcgov/cas-ciip-portal/pull/621
-  const productIsPublished = (formData, query) => {
-    const product = query.allProducts.edges.find(
-      ({node}) => node.rowId === formData.productRowId
-    )?.node;
-    if (product?.productState === 'PUBLISHED' || !product) return true;
-    return false;
-  };
+  const productIsPublished =
```

Comment:
Can we undo these changes? This file is going through major changes in #1580 & this is going to cause conflicts